app/src/main/javascript/bundles/experiment-page/src/results/ontologyIds.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
This is a great improvement over the not very nice and long conditional statements.
Well done! 👍 💯